pkg/docker/config/config.go

```diff
@@ -50,10 +50,17 @@ var (
 	ErrNotLoggedIn = errors.New("not logged in")
 	// ErrNotSupported is returned for unsupported methods
 	ErrNotSupported = errors.New("not supported")
+	// ErrNamespacedRegistry is returned if namespaced registries are not
+	// supported for the credential helper. Only the
+	// sysregistriesv2.AuthenticationFileHelper supports namespaced registries.
```

Comment:
```suggestion
	// ErrNamespacedRegistry is returned if storing credentials for namespaced registries
	// is not supported on the current system.
```
(modulo the name for “namespaced registries”), please.

Let’s make no promises about supporting them _only_ in files, or we would break that if adding more mechanisms.

Reply:
Fixed as suggested